packages/typescript-resolver-files-e2e/src/test-add-option/schema-prepend/types.generated.ts

```diff
@@ -22,7 +22,7 @@ export type Incremental<T> =
     };
 /** All built-in and custom scalars, mapped to their actual values */
 export type Scalars = {
-  ID: { input: string | number; output: string };
+  ID: { input: string; output: string | number };
```

Comment:
Previously, we didn't have a field with `ID` value in the schema.
This causes `Scalars.ID` here to use the default values [here](https://github.com/dotansimha/graphql-code-generator/blob/f28ae31a7b129acd193d59a01690e68b4da89ba9/packages/plugins/other/visitor-plugin-common/src/scalars.ts#L4-L7).
Note that a previous codegen `typescript` plugin version had the default ID value as: 
```
ID: { input: string | number; output: string };
```
That is why it's printed like this.

This PR updates the schema files to have at least one scalar to ensure config is reflected correctly